acceptance/suites/tests/030-tk-auth/010-tk-auth-setup.rb

```diff
@@ -45,7 +45,7 @@
   end
 
   step 'restart puppetserver' do
-    on(master, 'service puppetserver restart') 
+    on(master, 'service #{options[:puppetservice]} restart') 
```

Comment:
In order for interpolation to work the outer single-quotes need to be double-quotes too